tools/Customization/DevHome.Customization/Views/FileExplorerPage.xaml.cs

```diff
@@ -1,23 +1,83 @@
-// Copyright (c) Microsoft Corporation.
-// Licensed under the MIT License.
-
-using DevHome.Common.Extensions;
-using DevHome.Customization.ViewModels;
-using Microsoft.UI.Xaml;
-using Microsoft.UI.Xaml.Controls;
-
-namespace DevHome.Customization.Views;
-
-public sealed partial class FileExplorerPage : Page
-{
-    public FileExplorerViewModel ViewModel
-    {
-        get;
-    }
-
-    public FileExplorerPage()
-    {
-        ViewModel = Application.Current.GetService<FileExplorerViewModel>();
-        this.InitializeComponent();
-    }
-}
+// Copyright (c) Microsoft Corporation.

```

Comment:
Suggest reverting the line endings on this page.

Reply:
Oops did not notice that the initial commit changed the line endings😄. Reverted the line endings in latest commit. Thanks!